pkgs/games/fallout-ce/default.nix

```diff
@@ -0,0 +1,48 @@
+{ cmake
+, lib
+, SDL2
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation (finalAttr: {
+  pname = "fallout-ce";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "alexbatalov";
+    repo = "fallout1-ce";
+    rev = "v${finalAttr.version}";
+    hash = "sha256-EvRkOlvtiVao63S0WRKKuHlhfkdTgc0m6GTyv4EfJFU=";
+  };
+
+  fpattern = fetchFromGitHub {
```

Comment:
same as fallout2-ce, we can share this between the packages

Reply:
packaged